src/odemis/driver/simcam.py

```diff
@@ -289,22 +317,50 @@ def _simulate(self):
         binning = self.binning.value
         res = self.resolution.value
         pxs_pos = self.translation.value
-        shape = self._img.shape
+
+        #Shift the image based on the stage position
+        if self._stage:
+            pos = self._stage.position.value["x"], self._stage.position.value["y"]
+
+            phys_shift = self._orig_stage_pos[0] - pos[0], self._orig_stage_pos[1] - pos[1]
+            phys_shift = tuple(phys_shift)
+
+            # #convert to pxl
+            pxs = self._metadata[model.MD_PIXEL_SIZE]
+            px_shift = phys_shift[0] / pxs[0], phys_shift[1] / pxs[1]
+
+            npos=pxs_pos[0]+px_shift[0], pxs_pos[1]+px_shift[1]
+            pxs_pos=tuple(npos)
```

Comment:
Just write `pxs_pos = pxs_pos[0] + px_shift[0], pxs_pos[1] + px_shift[1]`
BTW, please make sure to leave spaces, as it's the official formatting. (PEP8)